src/components/Editor/Tabs.js

```diff
@@ -187,7 +185,7 @@ class Tabs extends PureComponent<Props, State> {
     return (
       <div className="source-header">
         {this.renderStartPanelToggleButton()}
-        {this.renderTabs()}
+        <div className="source-tabs">{this.renderTabs()}</div>
```

Comment:
Encapsulated `{}` feel better in separate lines for me as they are easier to absorb when reading.